content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -25,13 +25,14 @@ kubectl get csm --all-namespaces
 
 #### SDC Deployment for Operator
 - This feature deploys the sdc kernel modules on all nodes with the help of an init container.
+- Powerflex can be deployed with or without SDC. SDC deployment can be enabled and disabled by setting `X_CSI_SDC_ENABLED` value in CR file. By default, driver is deployed with SDC enabled.
```

Comment:
If disabled, does it mean that the SDC must already be installed? Maybe `Powerflex can be deployed with or without SDC.` should be `CSI Powerflex can be deployed with or without installing the SDC. If the SDC is already installed, this feature can be disabled.`

Reply:
> If disabled, does it mean that the SDC must already be installed? Maybe `Powerflex can be deployed with or without SDC.` should be `CSI Powerflex can be deployed with or without installing the SDC. If the SDC is already installed, this feature can be disabled.`

I think the feature is to make SDC optional for powerflex, so that even if SDC is not installed in any form, powerflex can be installed. In the case there's no SDC, powerflex will be functioning with NFS.